library/billing/src/Claim.php

```diff
@@ -1,36 +1,22 @@
 <?php
-/** @package OpenEMR
```

Comment:
To fully convert this class to psr2 standard
1. For all the variables at the top, change var to public.
(using public makes sense for now, since we don't know which variables are being directly accessed in the code, in order to avoid bringing in bugs)
2. For all the functions (except for the 2 private ones you added), change `function` to `public function`. AS above, using public to avoid bringin bugs)
(at some point can figure out whether vars and functions are private by checking them with phpstorm, but can get to that later after get this in the codebase)